### PR TITLE
Hw9

### PR DESCRIPTION
**Pull request by psp2003**

### CODE REVIEW

**Comment by psp2003** [comment]
2.3 2.4 - не совсем понятно задание

File: 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	Add to /etc/fstab
+	/dev/sdc2       swap    swap    defaults        0 0
+	[root@vm01 vit]# swapon -a
+	[root@vm01 vit]# swapon -s
+	Filename                                Type            Size    Used    Priority
+	/dev/dm-1                               partition       839676  0       -2
+	/dev/sdc2                               partition       524284  0       -3
+1.5. Configure the newly created XFS file system to persistently mount at /backup
+	Add to /etc/fstab
+	/dev/sdc1 /mount        xfs     defaults        0 0
```
Comment (merovigen) [inline 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt]: Should be /backup.

File: 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	└─sda2                 8:2    0    7G  0 part 
+	  ├─centos_vm01-root 253:0    0  6.2G  0 lvm  /
+	  └─centos_vm01-swap 253:1    0  820M  0 lvm  [SWAP]
+	sdb                    8:16   0    8G  0 disk 
+	└─sdb1                 8:17   0  300M  0 part /mnt/part1
+	sdc                    8:32   0    5G  0 disk 
+	├─sdc1                 8:33   0    2G  0 part /backup
+	└─sdc2                 8:34   0  512M  0 part [SWAP]
+	sr0                   11:0    1 1024M  0 rom 
+1.6. Configure the newly created swap space to be enabled at boot
```
Comment (merovigen) [inline 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt]: There is no new entry in fstab for new swap partition

File: 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt (hunk 1 above)
Comment (psp2003) [reply 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt]: fixed

File: 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt (hunk 2 above)
Comment (psp2003) [reply 9. Partitions, LVM/HW9.1 Malasherifov Vitalii.txt]: fixed

**Review by merovigen — CHANGES_REQUESTED** [review]
Show an entry for new swap partition in fstab, complete 2.6-2.7

**Review by merovigen — CHANGES_REQUESTED** [review]
Please remove 7 and 8 tasks from this PR.

**Pull request merged**